examples/example-react-native-wallet/App.tsx

```diff
@@ -0,0 +1,133 @@
+import {Keypair} from '@solana/web3.js';
+import React from 'react';
```

Comment:
nit: Maybe in the future we can refactor this into
`import React, { useState, useEffect } from 'react';
`
this way we can avoid writing `React.` before each hook call

Reply:
yeah for sure.